invokeai/app/invocations/denoise_latents.py

```diff
@@ -498,6 +499,33 @@ def parse_controlnet_field(
                 )
             )
 
+    @staticmethod
+    def parse_t2i_adapter_field(
+        exit_stack: ExitStack,
```

Comment:
`exit_stack` is unused.

Reply:
Later in version with tiled decode will be used, also trying to have same args for each field parsing.
But if you insist, I can remove it for now.